src/components/cart/modalContents/modalContents.js

```diff
@@ -0,0 +1,61 @@
+import { CartItems } from "./cartItems.js";
+import { EmptyCart } from "./emptyCart.js";
+
+export const ModalContents = ({ cartItems = [] }) => {
+  console.log(cartItems);
+  return `
+    ${
+      cartItems.length > 0
+        ? `
+         <!-- 전체 선택 섹션 -->
+          <div class="p-4 border-b border-gray-200 bg-gray-50">
+            <label class="flex items-center text-sm text-gray-700">
+              <input type="checkbox" id="cart-modal-select-all-checkbox" class="w-4 h-4 text-blue-600 border-gray-300 rounded focus:ring-blue-500 mr-2">
+              전체선택 (2개)
+            </label>
+          </div>
+        <!-- 아이템 목록 -->
+        <div class="flex-1 overflow-y-auto">
+            <div class="p-4 space-y-4">
+                ${cartItems.map((cartItem) => `${CartItems(cartItem)}`).join("")}
+            </div>
+        </div>
+        <!-- 하단 액션 -->
+        <div class="sticky bottom-0 bg-white border-t border-gray-200 p-4">
+          <!-- 선택된 아이템 정보 -->
+          <div class="flex justify-between items-center mb-3 text-sm">
+            <span class="text-gray-600">선택한 상품 (1개)</span>
+            <span class="font-medium">440원</span>
+          </div>
+          <!-- 총 금액 -->
+          <div class="flex justify-between items-center mb-4">
+            <span class="text-lg font-bold text-gray-900">총 금액</span>
+            <span id="cart-modal-total-price" class="text-xl font-bold text-blue-600">${cartItems
+              .reduce((acc, cartItem) => acc + cartItem.lprice * cartItem.quantity, 0)
```

Comment:
reduce 의 주체가 cartsItems 임을 이미 알고 있기 때문에 내부에서 x 로 축약해도 보는데 큰 문제가 없을거에유

```suggestion
              cartsItems.reduce((acc, x) => acc + x.lprice * x.quantity, 0)
```